jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/Vessel.java

```diff
@@ -57,4 +58,9 @@ public class Vessel {
 
   @Column(name = "dimension_unit", length = 3)
   private String dimensionUnit;
+
+  @Override
+  public boolean isNew() {
+    return this.id == null;
+  }
```

Comment:
I thought this was the default behaviour - What do we gain from implementing `Persistable` in this case?

Reply:
You're right, it's not required. I thought it was. Removed it.